Yoast/wordpress-seo
# Adds suggested plugins notifications

## Summary

This PR can be summarized in the following changelog entry:

* Adds notifications when we detect certain plugins are installed to suggest our AMP and ACF glue plugins.

## Test instructions

This PR can be tested by following these steps:

* Install ACF and/or AMP.
* Go to SEO -> Dashboard and notice (depending on which plugin(s) you installed in the previous step) that one or more new notifications have been added.
* Install the suggested plugins and check that the notifications disappear.
* Remove one of the suggested plugins and then remove AMP or ACF.
* Notice that there is only one notification still present for the plugin that still remains.

Fixes #7088
